node-operators/networks/collators/activities.md

```diff
@@ -1,72 +1,19 @@
-    | Minimum self-bond amount  |     {{ networks.moonbeam.staking.min_can_stk }} GLMR     |
-    | Minimum total bond amount |     {{ networks.moonbeam.staking.min_col_stk }} GLMR     |
-    |      Active set size      | {{ networks.moonbeam.staking.max_candidates }} collators |
-
-=== "Moonriver"
-    |         Variable          |                           Value                           |
-    |:-------------------------:|:---------------------------------------------------------:|
-    | Minimum self-bond amount  |     {{ networks.moonriver.staking.min_can_stk }} MOVR     |
-    | Minimum total bond amount |     {{ networks.moonriver.staking.min_col_stk }} MOVR     |
-    |      Active set size      | {{ networks.moonriver.staking.max_candidates }} collators |
-
-=== "Moonbase Alpha"
-    |         Variable          |                          Value                           |
-    |:-------------------------:|:--------------------------------------------------------:|
-    | Minimum self-bond amount  |     {{ networks.moonbase.staking.min_can_stk }} DEV      |
-    | Minimum total bond amount |     {{ networks.moonbase.staking.min_col_stk }} DEV      |
-    |      Active set size      | {{ networks.moonbase.staking.max_candidates }} collators |
-
-### Account in Polkadot.js {: #account-in-polkadotjs } 
-
-A collator has an account associated with its collation activities. This account is mapped to an author ID to identify the collator as a block producer and send the payouts from block rewards. 
-
-Depending on which network you want to run a collator node for, you can head to Polkadot.js for [Moonbeam](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/accounts){target=_blank}, [Moonriver](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.moonriver.moonbeam.network#/accounts){target=_blank}, or [Moonbase Alpha](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/accounts){target=_blank}. From there, you have two ways of proceeding in regard to having an account:
-
- - Importing an existing (or create a new) H160 account from external wallets or services such as [MetaMask](/tokens/connect/metamask/) and [MathWallet](/tokens/connect/mathwallet/)
- - Create a new H160 account with [Polkadot.js](/tokens/connect/polkadotjs/)
-
-Once you have an H160 account imported to Polkadot.js, you should see it under the **Accounts** tab. Make sure you have your public address at hand (`PUBLIC_KEY`), as it is needed to configure your [full node](/node-operators/networks/run-a-node/overview/) with the collation options.
-
-![Account in Polkadot.js](/images/node-operators/networks/collators/collator-polkadotjs-1.png)
+This guide will take you through the important timings to be aware of when leaving or reducing your self-bond amount, how to join and leave the candidate pool, and adjust your self-bond.
 
 ## Collator Timings {: #collator-timings }
```

Comment:
Add a round length to the table, so they can do the math

Reply:
I also think the !!!note after this table is too long. I would just only leave the part of value might change in the future, and take most of it out of the note